src/components/code/CodeBlock.jsx

```diff
@@ -3,15 +3,15 @@
 import React from 'react'
 import Highlight, { defaultProps } from 'prism-react-renderer'
 import CopyButton from '../button/CopyButton'
-import guidelines from 'zeppelin-element-library/guidelines.json'
+import guidelines from '@zlab-de/zel-react/guidelines.json'
```

Comment:
Whats the difference between this guidelines.json files, shouldn't they be equal?